src/main/java/org/embl/mobie/viewer/ui/MoBIELookAndFeelToggler.java

```diff
@@ -0,0 +1,33 @@
+package org.embl.mobie.viewer.ui;
+
+import com.formdev.flatlaf.FlatLightLaf;
+
+import javax.swing.*;
+
+public class MoBIELookAndFeelToggler {
+
+    private static LookAndFeel before;
+
+    public static void storeCurrentLaf() {
+        before = UIManager.getLookAndFeel();
+    }
+
+    public static void setMoBIELaf() {
+        FlatLightLaf.install();
```

Comment:
@K-Meech I was actually thinking that here (before line 16) you _always_  call the `storeCurrentLAF` before changing to the MoBIE one. Then one does not have to think about doing it explicitly. What do you think?

Reply:
Yes - that makes sense! I'll change it.

Reply:
done!